src/core/openthread-core-default-config.h

```diff
@@ -392,11 +392,11 @@
 /**
  * @def OPENTHREAD_CONFIG_6LOWPAN_REASSEMBLY_TIMEOUT
  *
- * The 6LoWPAN fragment reassembly timeout in seconds.
+ * The maximum timeout between 6LoWPAN fragments in seconds.
  *
  */
 #ifndef OPENTHREAD_CONFIG_6LOWPAN_REASSEMBLY_TIMEOUT
-#define OPENTHREAD_CONFIG_6LOWPAN_REASSEMBLY_TIMEOUT 5
+#define OPENTHREAD_CONFIG_6LOWPAN_REASSEMBLY_TIMEOUT 2
```

Comment:
Is this change intentional? If so, can you provide some reasoning?

Reply:
Yes. I changed it intentional. Because I refreshed the reassembly timer in the line 1426 of `mesh_forwarder.cpp`, so the fragment reassembly timeout and fragment priority entry timeout could use the same logic.

Reply:
Any reason why this value should be different than the 5 seconds currently configured for `kMaxLifeTime`?

Reply:
It seems I wrote the wrong comments. The comments in line  395 should be changed to `The reassembly timeout between 6LoWPAN fragments in seconds`. It is not the `maximum timeout`. Then `kMaxLifeTime` is the maximum timeout. `OPENTHREAD_CONFIG_6LOWPAN_REASSEMBLY_TIMEOUT` is the configured timeout.

Reply:
@zhanglongxia, that makes more sense. I applied the minor edit. Thanks.